src/cli.c

```diff
@@ -30,25 +33,42 @@ typedef struct {
 extern int errno;
 #endif
 
-static inline void noop() {};
 
 #define HELP_TEXT "usage: storj [<options>] <command> [<args>]\n\n"     \
     "These are common Storj commands for various situations:\n\n"       \
-    "setting up users profiles\n"                                       \
+    "setting up users profiles:\n"                                      \
+    "=========================\n"                                       \
     "  register                  setup a new storj bridge user\n"       \
     "  import-keys               import existing user\n"                \
     "  export-keys               export bridge user, password and "     \
     "encryption keys\n\n"                                               \
-    "working with buckets and files\n"                                  \
+    "working with buckets and files:\n"                                 \
+    "==============================\n"                                  \
     "  list-buckets\n"                                                  \
+    "  ls (lists the available buckets)\n"                              \
     "  list-files <bucket-id>\n"                                        \
+    "  ls <bucket-name> (lists the files in a bucket)\n"                \
+    "  get-bucket-id <bucket-name>\n"                                   \
     "  remove-file <bucket-id> <file-id>\n"                             \
-    "  add-bucket <name> \n"                                            \
+    "  rm <bucket-name> <file-name> (to remove a file from a bucket)\n" \
     "  remove-bucket <bucket-id>\n"                                     \
-    "  list-mirrors <bucket-id> <file-id>\n\n"                          \
-    "downloading and uploading files\n"                                 \
+    "  rm <bucket-name> (to remove a bucket)\n"                         \
+    "  add-bucket <name> \n"                                            \
+    "  mkbkt <bucket-name> \n"                                          \
+    "  list-mirrors <bucket-id> <file-id>\n"                            \
+    "  lm <bucket-name> <file-name>\n\n"                                \
+    "uploading files:\n"                                                \
+    "===============\n"                                                 \
     "  upload-file <bucket-id> <path>\n"                                \
-    "  download-file <bucket-id> <file-id> <path>\n"                    \
+    "  cp [-rR] <path-to-local-file-name> "                             \
```

Comment:
Grouping the unix style commands in the help text I think would be useful (example from comment at https://github.com/Storj/libstorj/pull/416#issuecomment-363872568)

Reply:
fixed the parenthesis and grouped the linux style commands